kafka-ui-api/src/main/java/com/provectus/kafka/ui/controller/AccessController.java

```diff
@@ -0,0 +1,36 @@
+package com.provectus.kafka.ui.controller;
+
+import com.provectus.kafka.ui.model.rbac.Role;
+import com.provectus.kafka.ui.service.rbac.AccessControlService;
+import java.util.Collection;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Mono;
+
+@RestController
+public class AccessController {
+
+  private final AccessControlService accessControlService;
+
+  @Autowired
+  public AccessController(AccessControlService accessControlService) {
+    this.accessControlService = accessControlService;
+  }
+
+  @RequestMapping("/getPermissions")
```

Comment:
/permissions ?

Reply:
Why not in openapi spec?

Reply:
it was a hand-written controller for some tests originally
fixed :)